test/zdtm.py

```diff
@@ -1886,8 +1886,8 @@ def do_run_test(tname, tdesc, flavs, opts):
                 check_visible_state(t, s, opts)
                 if opts['join_ns']:
                     check_joinns_state(t)
-                t.stop()
```

Comment:
This is here intentionally. Processes have to be fully functional after restore.